docs/Build.md

```diff
@@ -0,0 +1,72 @@
+Naggum Build Guide
+==================
+
+Windows
+-------
+
+To build Naggum under Windows, just use Visual Studio or MSBuild like that:
+
+    $ cd naggum
+    $ nuget restore
+    $ msbuild /p:Configuration=Release Naggum.sln
+
+Linux
+-----
+
+See general build instructions for Linux in the file `.travis.yml` inside of the
+Naggum root source directory.
+
+You'll need [Mono][mono], [NuGet][nuget] and [F# Compiler][fsharp] installed in
+you current environment. Some of them may or may not be part of your Mono
+installation; just make sure you have them all properly installed.
+
+Below is an example of setting up these tools on [NixOS Linux][nixos]; feel free
+to add instructions for any other non-standard distributives.
+
+### NixOS Linux
+
+First of all, install `mono` and `fsharp` packages to your environment:
+
+    $ nix-env -i mono fsharp
+
+Please note that you need `fsharp` >= 4.0 to build Naggum, which is currently on
+unstable NixOS channel. So maybe you'll need to switch to the unstable channel
+or build `fsharp` from [Nixpkgs][nixpkgs] source.
+
+According to the recommendations in
+[`patch-fsharp-targets`][patch-fsharp-targets] build helpers, you may want to
+set up the environment variable `FSharpTargetsPath` either globally or locally:
+
+    $ export FSharpTargetsPath=$(dirname $(which fsharpc))/../lib/mono/4.5/Microsoft.FSharp.Targets
+
+You'll need a NuGet executable. Unfortunately, there is no NixOS package for
+NuGet, so you need to download it manually:
```

Comment:
I'd use "have to" instead of "need to" to 1) avoid tautology; 2) make it clear that it's the requirement imposed on you, not your own desire :)